idkit/src/components/IDKitWidget/States/ErrorState.tsx

```diff
@@ -25,11 +43,11 @@ const ErrorState = () => {
 			</div>
 			<div>
 				<p className="text-center text-2xl font-semibold text-gray-900 dark:text-white">
-					{ERROR_TITLES[errorState?.code ?? ErrorCodes.GENERIC_ERROR]}
+					{(errorState?.code && ERROR_TITLES[errorState.code]) || ERROR_TITLES[AppErrorCodes.GenericError]}
```

Comment:
this should be equivalent yes?
```suggestion
					{(ERROR_TITLES[errorState?.code ?? AppErrorCodes.GenericError])}
```